python/langwatch/src/langwatch/telemetry/tracing.py

```
@@ -0,0 +1,628 @@
+from logging import warn
+from types import ModuleType
+from uuid import UUID
+import httpx
+import threading
+from deprecated import deprecated
+from langwatch.attributes import AttributeName
+from langwatch.utils.transformation import SerializableWithStringFallback, convert_typed_values
+from opentelemetry import trace as trace_api
+from opentelemetry.sdk.trace import TracerProvider
+from typing import Dict, List, Literal, Optional, Callable, Any, Type, TypeVar, Union, cast, TYPE_CHECKING
+from warnings import warn
```

Ruff (0.8.2)

Redefinition of unused `warn` from line 3

(F811)